views/static/login.php

```diff
@@ -1,17 +1,47 @@
+<?php
+
+require_once('connection.php');
+$link = Db::getInstance();
+
+//3. If the form is submitted or not.
+//3.1 If the form is submitted
+if (!isset($_SESSION['emailAddress'])) {
+//3.1.1 Assigning posted values to variables.
+    $emailAddress = $_POST['emailAddress'];
```

Comment:
youre not checking if your posts are set before checking for them

Reply:
i meant to say checking if they are set before grabbing them

Reply:
Will work on that later. I would like to post the overall authentication architecture.